docs/source/usersguide/depletion.rst

```diff
@@ -56,11 +36,40 @@ time::
     results = openmc.deplete.Results("depletion_results.h5")
     time, keff = results.get_keff()
 
-Note that the coupling between the transport solver and the transmutation solver
-happens in-memory rather than by reading/writing files on disk.
+Note that the coupling between the reaction rate solver and the transmutation
+solver happens in-memory rather than by reading/writing files on disk. OpenMC has two categories of transport operators for obtaining transmutation reaction
```

Comment:
```suggestion
solver happens in-memory rather than by reading/writing files on disk. OpenMC
has two categories of transport operators for obtaining transmutation reaction
```